packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganization.tsx

```diff
@@ -47,7 +47,7 @@ export const CreateOrganizationModal = (props: CreateOrganizationProps): JSX.Ele
   };
 
   return (
-    <Route path='createOrganization'>
+    <Route path='create-organization'>
```

Comment:
Why do we need this change?

Reply:
@nikosdouvlis basically this is for convention reasons(see `sign-in`). Is this breaking? I don't think so